examples/demo/src/visitors/VisitorEdit.js

```diff
@@ -45,7 +46,11 @@ const VisitorEdit = ({ classes, ...props }) => (
                     fullWidth={true}
                     formClassName={classes.email}
                 />
-                <DateInput source="birthday" />
+                <DateInput source="birthday" formClassName={classes.birthday} />
+                <PasswordInput
```

Comment:
No, the password should never be edited like this in an admin. An admin should not be able to see a user password. In fact, consider that the password is stored hashed, so it's not editable. The API should never expose the password, and the admin should use a password field to create a new password for an existing user.